analyzer/codechecker_analyzer/analyzers/clangsa/ctu_autodetection.py

```diff
@@ -171,6 +171,23 @@ def is_ctu_capable(self):
         return invoke_binary_checked(tool_path, ['-version'], self.environ) \
             is not False
 
+    @property
+    def extdef_mapping_can_read_pch(self):
+        """
+        Detects if the current clang-extdef-mapping can read PCH (.ast) files.
+        """
+
+        tool_path = self.mapping_tool_path
+
+        if not tool_path:
+            return False
+
+        help = invoke_binary_checked(tool_path, ['-help'], self.environ)
+        LOG.debug("output of clang-extdef-mapping\n %s" % help)
+        if not help:
+            return False
```

Comment:
This is not a hill I intend to die on, but this kind of implies that the binary is broken, right? Shouldn't this warrant an error, or raising an exception?